internal/impl/aws/input_kinesis.go

```diff
@@ -722,6 +682,65 @@ func (k *kinesisReader) maxShardRunBalancedConsumer(wg *sync.WaitGroup, streamID
 	return numShardClaimed, nil
 }
 
+// stealShard() attempts to steal shards if it meets the following condition:
+// * It is not its own clientId
+// * It has not reached the max shard number this client can claim
+// * The client it is stealing from has 2 or more shards more than this client currently has
+// It returns nil if it succesfully steals a shard, otherwise returns error errShardNotStolen if shards are not stolen or any errors if it encountered any
+func (k *kinesisReader) stealShard(wg *sync.WaitGroup, streamID string, clientClaims map[string][]awsKinesisClientClaim) ( error) {
```

Comment:
```suggestion
func (k *kinesisReader) stealShard(wg *sync.WaitGroup, streamID string, clientClaims map[string][]awsKinesisClientClaim) error {
```

I'm sorry but formatting is pretty serious in Go, lol. I'm pretty sure our PR will be rejected by Benthos if formatting is not right that's why I'm picky. I think you need to setup your go formatter in your IDE.

Reply:
hmm i already have setup my goformatter to use goimport though

Reply:
hmm, strange. It may be not setup to run on save?!

Reply:
ahhh i think it was silently failing on vscode :( i fixed it now, hopefully no more 😅